### PR TITLE
feature[TW27211]: Add create version endpoint

### PR DESCRIPTION
**Pull request by audrey-denk**
Change-Id: Ib7b04326b3aee6ad52eb81b81f56b45dd74b3e92

### CODE REVIEW

File: plugins/org.eclipse.osee.ats.api/src/org/eclipse/osee/ats/api/config/AtsConfigEndpointApi.java [hunk 1]
```diff
@@ -137,6 +138,12 @@ public interface AtsConfigEndpointApi {
    @Produces(MediaType.APPLICATION_JSON)
    public Version getVersion(@PathParam("verId") ArtifactId verId);
 
+   @POST
+   @SkipAtsConfigJsonWriter
```
Comment (dondunne) [inline plugins/org.eclipse.osee.ats.api/src/org/eclipse/osee/ats/api/config/AtsConfigEndpointApi.java:142]: Remove the "/create".  That is assumed by POST.  The method can still be createVersion.

File: plugins/org.eclipse.osee.ats.core/src/org/eclipse/osee/ats/core/version/AtsVersionServiceImpl.java [hunk 2]
```diff
@@ -327,6 +327,16 @@ public Collection<IAtsVersion> getVersions(IAtsTeamDefinition teamDef) {
    public Version createVersion(String title, IAtsChangeSet changes) {
       return createVersion(title, Lib.generateArtifactIdAsInt(), changes);
    }
+   
+   @Override
+   public Version createVersion(String name, String description, ArtifactId teamId) {
+      IAtsChangeSet changes = atsApi.createChangeSet("Create Version: "+name);
+      Version newVersion = createVersion(name,changes);
+      newVersion.setDescription(description);
```
Comment (dondunne) [inline plugins/org.eclipse.osee.ats.core/src/org/eclipse/osee/ats/core/version/AtsVersionServiceImpl.java:335]: I'd load the teamId and make sure it exists and isOfType AtsArtifactTypes.TeamDefinition.  Exception if it doesn't.

**Review by david-w-miller — COMMENTED** [review]
Quick checks - since anything could be coming in, can we check to make sure the name and description are reasonable?

File: plugins/org.eclipse.osee.ats.api/src/org/eclipse/osee/ats/api/config/AtsConfigEndpointApi.java (hunk 1 above)
Comment (audrey-denk) [reply plugins/org.eclipse.osee.ats.api/src/org/eclipse/osee/ats/api/config/AtsConfigEndpointApi.java:142]: I thought of that but in some cases we are explicit.  I'll update it.

**Review by dondunne — APPROVED** [review]

**Review by david-w-miller — APPROVED** [review]

**Pull request merged**